src/components/dataset/new.vue

```diff
@@ -130,8 +132,19 @@ const hideOrComplete = () => {
 
 <style lang="scss">
 @import '../../assets/scss/variables';
+@import '../../assets/scss/mixins';
 
 #dataset-new-success {
+  display: flex;
+  align-items: center;
+  margin-bottom: 7px;
+
+  > div {
+    @include text-block;
```

Comment:
Could this `<div>` be a `<p>` instead? Then `text-block` would be applied automatically.